app/test_main.py

```diff
@@ -1,3 +1,39 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, goals",
+    [pytest.param(14, 14, [0, 0],
+                  id="should return first 15 dog and cat years"),
+     pytest.param(28, 29, [3, 3],
+                  id="should return every 5 next dog and cat years"),
+     pytest.param(27, 28, [2, 2],
+                  id="should return every 5 next dog and cat years"),
```

Comment:
The expected result `[2, 2]` for `cat_age=27` and `dog_age=28` seems incorrect. According to the conversion rules, 27 cat years should convert to 3 human years (15 + 9 + 3), and 28 dog years should convert to 3 human years (15 + 9 + 4). Please verify this calculation.